src/components/ImportOnyxState/utils.ts

```diff
@@ -50,4 +54,42 @@ function cleanAndTransformState<T>(state: string): T {
     return transformedState;
 }
 
-export {transformNumericKeysToArray, cleanAndTransformState};
+const processStateImport = (transformedState: OnyxValues) => {
```

Comment:
```suggestion
function importState(transformedState: OnyxValues): Promise<void> {
```

could we rename to `importState`?